cloud/gcp-functions/cmd/orphanedNeg.go

```diff
@@ -19,13 +19,13 @@ func CleanOrphanedNEG(w http.ResponseWriter, r *http.Request) {
 	// Initialize the client for Network Endpoint Groups, Backend Services, and getting Zones list.
 	negClient, err := compute.NewNetworkEndpointGroupsRESTClient(ctx)
 	if err != nil {
-		log.Fatalf("Failed to create NEG client: %v", err)
+		log.Fatal("Failed to create NEG client: %v", err)
```

Comment:
you need to use `Fatalf` not `Fatal`, otherwise `%v` won't be replaces by `err`.